### PR TITLE
Fix opcache preloading

### PR DESCRIPTION
**Pull request by piotrantosik**
This PR allows using opcache preloading function
Fix https://github.com/craue/CraueFormFlowBundle/issues/363

### CODE REVIEW

File: Event/FormFlowEventTrait.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    /**
+     * @return FormFlowInterface
+     */
+    public function getFlow()
+    {
+        return $this->flow;
+    }
+}
```
Comment (craue) [inline Event/FormFlowEventTrait.php:29]:
```suggestion
/**
 * @internal
 * @author Christian Raue <christian.raue@gmail.com>
 * @copyright 2011-2020 Christian Raue
 * @license http://opensource.org/licenses/mit-license.php MIT License
 */
trait FormFlowEventTrait {

	/**
	 * @var FormFlowInterface
	 */
	protected $flow;

	/**
	 * @param FormFlowInterface $flow
	 */
	public function __construct(FormFlowInterface $flow) {
		$this->flow = $flow;
	}

	/**
	 * @return FormFlowInterface
	 */
	public function getFlow() {
		return $this->flow;
	}

}
```

File: composer.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
 	"require-dev": {
 		"craue/translations-tests": "^1.0",
 		"doctrine/common": "~2.7",
+		"doctrine/dbal": "<3.0",
```
Comment (craue) [inline composer.json:35]: Please revert this change and rebase to current master. I've added support for DBAL 3.

**Review by craue — CHANGES_REQUESTED** [review]
Please also revert the permission change on `FormFlowEvent.php`.

**Comment by piotrantosik** [comment]
@craue I have made changes, please review again.

**Comment by craue** [comment]
Thank you, @piotrantosik.

**Pull request merged**